source/Firebase/Functions/ApiDefinition.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+using UIKit;
+using Foundation;
+using ObjCRuntime;
+using CoreGraphics;
+using Firebase.Core;
+
+namespace Firebase.Functions
+{
+	// @interface FIRFunctions : NSObject
+	[DisableDefaultCtor]
+	[BaseType (typeof (NSObject), Name = "FIRFunctions")]
+	interface Functions
+	{
+		// + (FIRFunctions *)functions;
+        [Static]
```

Comment:
Check tabs, we use [Mono Coding Guidelines][1]. You can set this in 
Settings of VS -> Source Code -> Code Formatting -> Policy -> Select Mono
[1]: https://www.mono-project.com/community/contributing/coding-guidelines/

Reply:
Thanks for the info. I will run formatting over all files when the changes are implemented. Don't want to mess up your change suggestions